app/views/liquids/_stackexchange.html.erb

```diff
@@ -1,14 +1,14 @@
 <div class="ltag__stackexchange--container">
   <div class="ltag__stackexchange--title-container">
-    <header>
+    <header id="ltag-stackexchange-header">
```

Comment:
I don't think this ID is being used anywhere? If it's not, then let's remove it. There could easily be more than one of these liquid tags in a given post, so an ID may not be appropriate anyway 🙂